enterprise/server/backends/kms/kms.go

```diff
@@ -82,6 +106,51 @@ func (k *KMS) initGCPClient(ctx context.Context) error {
 	return nil
 }
 
+type awsKMSARN struct {
```

Comment:
random non-actionable go style complaint, this is exactly why I wish this would be awsKmsArn